nexus/db-model/src/quota.rs

```diff
@@ -78,9 +80,9 @@ impl From<views::SiloQuotas> for SiloQuotas {
             silo_id: silo_quotas.silo_id,
             time_created: Utc::now(),
             time_modified: Utc::now(),
-            cpus: silo_quotas.cpus,
-            memory: silo_quotas.memory.into(),
-            storage: silo_quotas.storage.into(),
+            cpus: silo_quotas.limits.cpus,
+            memory: silo_quotas.limits.memory.into(),
+            storage: silo_quotas.limits.storage.into(),
         }
     }
 }
```

Comment:
question stands from the previous PR, is this From used? it doesn't make sense to go from view to model

Reply:
I deleted it and ran the quotas tests which passed so... no? I don't recall why I added that initially.